src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -1721,6 +1721,55 @@ public bool Equals(Program other)
 parameters: CSharp6Implicit);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
+        public async Task TestOverrideEqualsOnRefStructReturnsFalse()
+        {
+            await TestWithPickMembersDialogAsync(
+@"
+ref struct Program
+{
+    public string s;
+    [||]
+}",
+@"
+ref struct Program
+{
+    public string s;
+
+    public override bool Equals(object obj)
+    {
+        return false;
+    }
+}",
+chosenSymbols: null,
+parameters: CSharp6Implicit);
```

Comment:
why C# 6.  did we even have ref-structs in c# 6?

Reply:
Fixed, I removed the option entirely.